src/olympia/files/models.py

```diff
@@ -706,6 +712,50 @@ class Meta:
         db_table = 'validation_annotations'
 
 
+class WebextPermission(ModelBase):
```

Comment:
JSON in a database field is ... icky. Plus so much unnecessary duplication (admittedly relative to the amount of data already in the database, minimal).  Unless there's a performance hit I lean towards ~~over~~correct-engineering.